### PR TITLE
Additional update for AQMv7 o3/pm25 verification

### PR DESCRIPTION
**Pull request by Ho-ChunHuang-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

daily cronjob for prep, stats, and plots steps since February 13 2024

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

please see[ instruction](https://docs.google.com/document/d/1zpDRPxGMEE3j3VGqFHr8K4Kh15AtWhqq/edit?usp=sharing&ouid=110971079403051862772&rtpof=true&sd=true)

Please remember to add export COMINaqm in driver scripts as stated in the instruction

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**
YES
- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**
YES
- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: jobs/JEVS_AQM_PREP [hunk 1]
```diff
@@ -76,21 +75,20 @@ export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}
 export COMOUTproc=${COMOUTproc:-${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}}
 mkdir -p ${COMOUT} ${COMOUTproc}
 
-# alternative choice is dan option different from aqons
+# alternative choice is dan option different from aqobs
```
Comment (AliciaBentley-NOAA) [inline jobs/JEVS_AQM_PREP:78]: @Ho-ChunHuang-NOAA What is a "dan option"?

File: parm/metplus_config/plots/aqm/grid2obs/py_plotting_ozmax8_headline.config [hunk 2]
```diff
@@ -116,27 +117,29 @@ export EVAL_PERIOD="TEST"
...
-export VALID_BEG="$PDYm31"
-export VALID_END="$VDATE"
-export INIT_BEG="$PDYm31"
-export INIT_END="$VDATE"
+
+export VALID_BEG="${PLOT_START}"
+export VALID_END="${PLOT_END}"
+
+export INIT_BEG="${PLOT_START}"
+export INIT_END="${PLOT_END}"
```
Comment (AliciaBentley-NOAA) [inline parm/metplus_config/plots/aqm/grid2obs/py_plotting_ozmax8_headline.config:125]: Why are we no longer using $PDYm31, etc.? It seems to me that some of these changes are beyond the scope of an AQMv7 upgrade. @Ho-ChunHuang-NOAA @ShelleyMelchior-NOAA @PerryShafran-NOAA

File: jobs/JEVS_AQM_PREP (hunk 1 above)
Comment (Ho-ChunHuang-NOAA) [reply jobs/JEVS_AQM_PREP:78]: typo, "an", will fix

File: parm/metplus_config/plots/aqm/grid2obs/py_plotting_ozmax8_headline.config (hunk 2 above)
Comment (Ho-ChunHuang-NOAA) [reply parm/metplus_config/plots/aqm/grid2obs/py_plotting_ozmax8_headline.config:125]: The default remains to be PDYm31 as PLOT_START defined in JEVS_AQM_PLOTS.  I think all values should be defined in the J-jobs but not hard-coded in scripts several layers down from J-jobs.  Also allow user an option to change time period in their own driver scripts for their own testing.

The original code setting is fine in operation (but all the file checking are unnecessary in operational environment by default).  But it runs in risk as if user set VDATE earlier than PDYm31, as I did before to reproduce graphic for presentation, the job will crash.

**Comment by Ho-ChunHuang-NOAA** [comment]
Per latest instruction from Alicia, can you remove this PR.  I do not think it need to remain open till EVSv2.0.

**Comment by AliciaBentley-NOAA** [comment]
@Ho-ChunHuang-NOAA I can close it for you now, if you would like! Do you have all your changes saved somewhere else?

**Comment by Ho-ChunHuang-NOAA** [comment]
@AliciaBentley-NOAA I will create a new branch from develop and keep PR456 branch for reference.  Thank you.

**Comment by AliciaBentley-NOAA** [comment]
@Ho-ChunHuang-NOAA Excellent. To double check with you- can I close this PR now?

**Comment by Ho-ChunHuang-NOAA** [comment]
@AliciaBentley-NOAA On the second thought, please keep it open.  It is easier to spot the needed changes from code changes. list

**Comment by AliciaBentley-NOAA** [comment]
@Ho-ChunHuang-NOAA Ok, we can keep it open! Can you please change the name of this PR, so that we don't get it confused with the next AQMv7 PR that you will submit?

**Comment by Ho-ChunHuang-NOAA** [comment]
@AliciaBentley-NOAA   I am done.  Please close this PR.  Thanks.

**Comment by AliciaBentley-NOAA** [comment]
Closing this PR so that another PR (with only the AQMv7 changes) can be submitted. 
CC @Ho-ChunHuang-NOAA @ShelleyMelchior-NOAA @PerryShafran-NOAA

**Pull request closed without merging**